docs/developer-guide/running-locally.md

```diff
@@ -4,9 +4,9 @@
 
 During development, it might be viable to run ArgoCD outside of a Kubernetes cluster. This will greatly speed up development, as you don't have to constantly build, push and install new ArgoCD Docker images with your latest changes.
 
-You will still need a working Kubernetes cluster, as described in the [contributing.md](Contribution Guide), where ArgoCD will store all of its resources.
+You will still need a working Kubernetes cluster, as described in the [contributing.md](contributing.md), where ArgoCD will store all of its resources.
```

Comment:
Actually, I think the two should be exchanged - so `[contributing.md](Contribution Guide)` should be `[Contribution Guide](contributing.md)`